README.md

```diff
@@ -8,140 +8,194 @@
+          @dpc-sdp/ripple-test-tools
+        </a>
+      </td>
+      <td>
+        <a href="https://www.npmjs.com/package/@dpc-sdp/ripple-test-tools">
+          <img src="https://img.shields.io/npm/v/@dpc-sdp/ripple-test-tools" alt="Version">
+        </a>
+      </td>
+      <td>
+        Cypress.io helper library for testing a ripple nuxt site and tide backend
+      </td>
+    </tr>
+
+  </tbody>
+</table>
 
 ### Built With
 
 * [Vue](https://vuejs.org/)
 * [Nuxt](https://nuxtjs.org)
-* [Storybook](https://storybook.js.org/)
-
-
-## Usage
 
-Ripple is a monorepo which contains several packages published to NPM
+### Browser support
 
-[@dpc-sdp/ripple-nuxt-tide](https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-tide) - Package for integrating a Tide backend into a Ripple site. Docs: [packages/ripple-nuxt-tide/README.md](https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-tide/README.md)
+Ripple components support all **modern browsers**, including IE11.
 
-[@dpc-sdp/ripple-nuxt-ui](https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-ui) - Adds Ripple component library to a Nuxt site, can be used independently of @dpc-sdp/ripple-nuxt-tide. Docs: [packages/ripple-nuxt-ui/README.md](https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-ui/README.md)
+## Usage
 
-Ripple Vue component library 
-[@dpc-sdp/ripple-global](https://www.npmjs.com/package/@dpc-sdp/ripple-global) - Each Ripple Vue component is published individually to NPM, See Ripple global for usage.
+Ripple components are published individually as npm packages and can be imported in any Vue project. See the [List of packages](https://github.com/dpc-sdp/ripple/blob/master/RIPPLE_COMPONENTS.md).
 
-### In a Nuxt application
+### Use without Tide Backend
 
-Ripple is primarily used to build websites using the [Nuxt](https://nuxtjs.org) framework.
+**Jump to [Use with Tide integration](#use-with-tide-integration) if you are going to use Ripple with SDP [Tide](https://github.com/dpc-sdp/tide) Backend**
 
-_For examples on how to use Ripple in a Nuxt application, please refer also to the [Documentation](https://dpc-sdp.github.io/sdp-docs/ripple/)_
+#### Use Ripple in Vue.js app
 
-#### With Tide
+Check out our example Vue.js app: [Vue app example](https://github.com/dpc-sdp/ripple/tree/master/examples/vue-example-app).
 
-If you are working with a Tide Drupal content backend you can add [@dpc-sdp/ripple-nuxt-tide](https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-tide), this adds `@dpc-sdp/ripple-nuxt-ui` as a dependency, no need to add it yourself.
+#### Use Ripple in Nuxt.js(SSR) app
 
-*Installation*
+For users are using Nuxt.js. You can use our Nuxt module [@dpc-sdp/ripple-nuxt-ui](https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-ui) to add Ripple components library to your Nuxt.js project. This configures `@dpc-sdp/ripple-global` and adds required webpack config.
```

Comment:
The first sentence might be redundant since you already mentioned in second sentence `...to your Nuxt.js project`.